src/modules/core/components/UserPickerWithSearch/UserPickerWithSearch.tsx

```diff
@@ -0,0 +1,249 @@
+import React, { ReactNode, useCallback, useRef } from 'react';
+import { defineMessages, MessageDescriptor, useIntl } from 'react-intl';
+import compose from 'recompose/compose';
+import classNames from 'classnames';
+
+import { useField } from 'formik';
+import { AnyUser } from '~data/index';
+import { Address, SimpleMessageValues } from '~types/index';
+import { getMainClasses } from '~utils/css';
+
+import { ItemDataType, WrappedComponentAdditionalProps } from '../OmniPicker';
+import { Props as WithOmnipickerInProps } from '../OmniPicker/withOmniPicker';
+import { InputLabel } from '../Fields';
+import Icon from '../Icon';
+import UserAvatar from '~core/UserAvatar';
+
+import styles from './UserPickerWithSearch.css';
+import { ItemDefault } from '~core/SingleUserPicker';
+import Dropdown from './Dropdown';
+import withAdditionalOmniPicker from '~core/OmniPicker/withAdditionalOmniPicker';
+import Button from '~core/Button';
+
+type AvatarRenderFn = (
+  address: Address,
+  user?: ItemDataType<AnyUser>,
+) => ReactNode;
+
+const MSG = defineMessages({
+  selectMember: {
+    id: 'SingleUserPicker.selectMember',
```

Comment:
```suggestion
    id: 'UserPickerWIthSearch.selectMember',
```

This applies to all message descriptor id's